test/test_core.py

```diff
@@ -8324,6 +8325,7 @@ def test_asyncify_main_module(self):
   def test_emscripten_lazy_load_code(self, conditional):
     if self.get_setting('STACK_OVERFLOW_CHECK'):
       self.skipTest('https://github.com/emscripten-core/emscripten/issues/16828')
+    self.set_setting('STACK_OVERFLOW_CHECK', 0)
```

Comment:
(here too)

Reply:
This is because LAZY_LOAD_CODE is broken with STACK_OVERFLOW_CHECK.. see the bug on the line above.